config/cmip5.yaml

```diff
@@ -4,10 +4,10 @@ translator: Cmip5Translator
 
 sources:
 
-  - metadata_yaml: /g/data/tm70/intake/metadata/cmip5_al33/metadata.yaml
+  - metadata_yaml: /g/data/xp65/admin/intake/metadata/cmip5_al33/metadata.yaml
     path:
       - /g/data/dk92/catalog/v2/esm/cmip5-al33/catalog.json
```

Comment:
```suggestion
      - /g/data/al33/catalog/v2/esm/catalog.json
```

Reply:
What's the difference between the two paths? Both `catalog.json` exist, and a `diff` shows no change.

I'm assuming that, because the timestamp on the `dk92` version is yesterday, that there's some reason things from `al33` are being repeatedly copied onto `dk92`; what's the background behind that?

Reply:
The catalogs are identical. My point is the standard way to access NCI maintained collection is to follow that pattern. The dk92 project is not an NCI data collection, fs38, al33, rr3 and oi10 are.